vignettes/datatable-sd-usage.Rmd

```diff
@@ -0,0 +1,258 @@
+---
+title: "Using .SD for Data Analysis"
+date: "`r Sys.Date()`"
+output:
+  rmarkdown::html_vignette:
+    toc: true
+    number_sections: true
+vignette: >
+  %\VignetteIndexEntry{Using .SD for Data Analysis}
+  %\VignetteEngine{knitr::rmarkdown}
+  \usepackage[utf8]{inputenc}
+---
+
+```{r, echo = FALSE, message = FALSE}
+require(data.table)
+knitr::opts_chunk$set(
+  comment = "#",
+    error = FALSE,
+     tidy = FALSE,
+    cache = FALSE,
+  collapse = TRUE,
+  fig.path = 'plots/',
+  out.width = '100%',
+  dpi = 144
+)
+```
+
+This vignette will explain the most common ways to use the `.SD` variable in your `data.table` analyses. It is an adaptation of [this answer](https://stackoverflow.com/a/47406952/3576984) given on StackOverflow.
+
+# What is `.SD`?
```

Comment:
https://rdatatable.gitlab.io/data.table/library/data.table/doc/datatable-sd-usage.html
rendered from R it result into `What is <code>.SD</code>` tab name in browser, maybe better remove code and leave .SD as plaintext